00-HTML-CSS-basics/HTML/1.II-1.III-mytodolist.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+    <!-- The Basics of HTML - Exercise 1.II and III -->
+    <head>
+        <meta charset="UTF-8">
+        <title>Ex 1.II and III - My todo list</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <meta name="keywords" content="Bootcamp, Startup,HTML">
+        <meta name="description" content="HTML exercises">
+        <meta name="author" content="Fernando Farina">
+    </head>
+    <body>
+        <h1>My todo list</h1>
+        <ol>
```

Comment:
I'd suggest to use `ul` instead of `ol` because is a todo list and not a list which is being ordered.